### DISCUSSION TITLE
Is there a reason why the 'executors' directory/package is not in this repository?

### DISCUSSION THREAD

**Discussion by alxfed** [agronholm/apscheduler#561] [Q&A] [answered]
Hi! I'm working on custom executor for asyncio version of your scheduler. It came to my attention that the 'executors' package / directory is not __here__, even though it is present in the pip archive (of course). May I ask why? I'm very curious.

**Comment by agronholm** [comment, marked as answer]
It is here, but you're probably looking at the master branch and not 3.x where the current stable release is being maintained. The master branch is where 4.0 development happens and a lot has changed there.

**Reply by alxfed** [reply]
Thank you. I didn't notice that you did it this way. Typically it's the opposite, development is on a branch other than main.
